korgan/reply_menu_handlers.py

```
@@ -0,0 +1,289 @@
+
+    if not context.strip() or len(context.strip()) < 80:
+        await state.update_data(mode="contract_details")
+        await message.answer(
+            "🤝 Опишите договор одним сообщением:\n"
+            "• какой договор нужен;\n"
+            "• кто стороны и их роли;\n"
+            "• предмет договора;\n"
+            "• цена/оплата, если есть;\n"
+            "• срок;\n"
+            "• важные особые условия.\n\n"
+            "Можно также прикрепить документы или переписку. KORGAN сам определит точный вид договора и проверит применимые нормы РК.",
```

Ruff (0.16.1)

[warning] String contains ambiguous `Р` (CYRILLIC CAPITAL LETTER ER). Did you mean `P` (LATIN CAPITAL LETTER P)?

(RUF001)